shared/message/behaviour/Subsumption.proto

```diff
@@ -21,6 +21,9 @@ syntax = "proto3";
 
 package message.behaviour;
 
+/**
+ *
+ */
```

Comment:
Subsumption is the system we use to grant different modules access to different limbs of the robot. Each module only requests the limbs that it needs to function and a priority for how important the module is.

This message appears to only be used for sending subsumption information to NUsight. Check out the [Controller module](https://github.com/NUbots/NUbots/tree/master/module/behaviour/Controller) for how this is used. 

@TrentHouliston can also provide details since he wrote the system.